Sources/Misc/Concurrency/Purchases+async.swift

```diff
@@ -47,6 +47,15 @@ extension Purchases {
         }
     }
 
+    func redeemWebPurchaseAsync(_ deepLink: Purchases.DeepLink.WebPurchaseRedemption) async throws
+    -> WebPurchaseRedemptionResult {
```

Comment:
I think the return type looks a bit weird because if the result is a `WebPurchaseRedemptionResult.Error` it would never actually be returned because it would throw first. But if it is successful you still need to check if the type `WebPurchaseRedemptionResult.Success`.

Reply:
Yeah... as I mentioned in the other comment, we're planning to add more result types so it would make more sense there... but yeah probably doesn't make sense for this to throw though since the error will be part of one of the result types. Will try to change that